vllm/worker/model_runner.py

```diff
@@ -172,8 +172,11 @@ def load_model(self) -> None:
             )
 
         self.model_memory_usage = m.consumed_memory
+        logger_data = {
+            "model_memory_usage": f"{self.model_memory_usage / float(2**30):.4f}"
+        }
```

Comment:
I think I'd suggest a variable for the model memory usage value:
```suggestion
        model_memory_usage_gb = self.model_memory_usage / float(2**30)
        logger_data = {
            "event": "model-loaded",
            "model_memory_usage_gb": model_memory_usage_gb,
        }
```

If there's a name for the model, I think that'd be useful context too.